app/src/main/java/io/github/androidacademyglobal/profile/ProfileViewModel.kt

```diff
@@ -0,0 +1,29 @@
+package io.github.androidacademyglobal.profile
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import io.github.androidacademyglobal.R
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.stateIn
+
+class ProfileViewModel : ViewModel() {
+    val state: StateFlow<ProfileScreenState> =
+        // just stub for this moment
```

Comment:
Would be better to move the stub onto repository level (create `*SomeRepoStub` class and it'll be self-documenting)